app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileViewModel.kt

```diff
@@ -133,9 +135,10 @@ class SelfUserProfileViewModel @Inject constructor(
         userProfileState = userProfileState.copy(statusDialogData = null)
     }
 
-    fun changeStatus(status: UserStatus) {
+    fun changeStatus(status: UserAvailabilityStatus) {
         setNotShowStatusRationaleAgainIfNeeded(status)
-        userProfileState = userProfileState.copy(status = status)
+        viewModelScope.launch {  updateStatus(status) }
```

Comment:
Check proper formatting 👀

Reply:
Maybe also add a TODO comment to add here the broadcast message to inform everyone about the self user new status?